src/client/client4.js

```diff
@@ -1631,6 +1631,15 @@ export default class Client4 {
         );
     };
 
+    markPostAsUnread = async (userId, postId) => {
+        this.trackEvent('api', 'api_post_set_unread_post');
+
+        return this.doFetch(
+            `${this.getUserRoute(userId)}/posts/${postId}/set_unread`,
```

Comment:
can I mark a channel as unread for another user? if not maybe we should consider using `me` here ?

Reply:
there was a discussion about it on the [api-doc PR](https://github.com/mattermost/mattermost-api-reference/pull/464#discussion_r311269544) that we might consider this at some point for admins.